### ISSUE TITLE
Experiencer2/core.lua:58: attempt to index local 'colors' (a nil value)

### ISSUE THREAD

**Issue by NoShotz** [DJScias/Experiencer2#5]
```
1470x Experiencer2/core.lua:58: attempt to index local 'colors' (a nil value)
[string "@Experiencer2/core.lua"]:58: in function `SetReputationColor'
[string "@Experiencer2/modules/reputation.lua"]:154: in function `SetStanding'
[string "@Experiencer2/modules/reputation.lua"]:874: in function `?'
[string "@Masque/Libs/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:109: in function <...que/Libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:109>
[string "=[C]"]: ?
[string "@Masque/Libs/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:19: in function <...que/Libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:15>
[string "@Masque/Libs/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:54: in function `Fire'
[string "@BagAppraiser/libs/AceEvent-3.0-4/AceEvent-3.0.lua"]:120: in function <...dOns/BagAppraiser/libs/AceEvent-3.0/AceEvent-3.0.lua:119>

Locals:
self = <table> {
 FormatNumber = <function> defined @Experiencer2/core.lua:274
 FinishedProgressColor = <function> defined @Experiencer2/core.lua:248
 GetProgressColor = <function> defined @Experiencer2/core.lua:240
 EnableModule = <function> defined @Masque/Libs/AceAddon-3.0/AceAddon-3.0.lua:328
 modules = <table> {
 }
 baseName = "Experiencer2"
 IterateEmbeds = <function> defined @Masque/Libs/AceAddon-3.0/AceAddon-3.0.lua:438
 SecureHookScript = <function> defined @BestInSlotRedux/Libs/AceHook-3.0/AceHook-3.0.lua:409
 SendModuleChatMessage = <function> defined @Experiencer2/core.lua:763
 ShowBar = <function> defined @Experiencer2/core.lua:226
 RawHookScript = <function> defined @BestInSlotRedux/Libs/AceHook-3.0/AceHook-3.0.lua:394
 SetSplits = <function> defined @Experiencer2/core.lua:958
 SetDefaultModulePrototype = <function> defined @Masque/Libs/AceAddon-3.0/AceAddon-3.0.lua:405
 defaultModuleState = true
 OpenContextMenu = <function> defined @Experiencer2/core.lua:1040
 IsEnabled = <function> defined @Masque/Libs/AceAddon-3.0/AceAddon-3.0.lua:447
 UpdateVisiblity = <function> defined @Experiencer2/core.lua:214
 DisableModule = <function> defined @Masque/Libs/AceAddon-3.0/AceAddon-3.0.lua:346
 ToggleTextVisilibity = <function> defined @Experiencer2/core.lua:782
 RegisterModule = <function> defined @Experiencer2/core.lua:189
 RegisterMessage = <function> defined @Masque/Libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:80
 UnregisterMessage = <function> defined @Masque/Libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:143
 GetHoveredModule = <function> defined @Experiencer2/core.lua:754
 FindValidModuleForBar = <function> defined @Experiencer2/core.lua:907
 IsHooked = <function> defined @BestInSlotRedux/Libs/AceHook-3.0/AceHook-3.0.lua:492
 db = <table> {
 }
 RefreshModule = <function> defined @Experiencer2/core.lua:313
 SetEnabledState = <function> defined @Masque/Libs/AceAddon-3.0/AceAddon-3.0.lua:420
 Hook = <function> defined @BestInSlotRedux/Libs/AceHook-3.0/AceHook-3.0.lua:273
 enabledState = true
 RegisterEvent = <function> defined @Masque/Libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:80
 IterateModules = <function> defined @Masque/Libs/AceAddon-3.0/AceAddon-3.0.lua:433
 RefreshBars = <function> defined @Experiencer2/core.lua:515
 RawHook = <function> defined @BestInSlotRedux/Libs/AceHook-3.0/AceHook-3.0.lua:312
 GetModuleFromModuleIndex = <function> defined @Experiencer2/core.lua:1346
 GetPlayerClassColor = <function> defined @Experiencer2/core.lua:52
 OnInitialize = <function> defined @Experiencer2/core.lua:84
 NewModule = <function> defined @Masque/Libs/AceAddon-3.0/AceAddon-3.0.lua:231
 ContextMenu = ExperiencerContextMenuFrame {
 }
 GetModule = <function> defined @Masque/Libs/AceAddon-3.0/AceAddon-3.0.lua:206
 GetModuleIndexFromMousePosition = <function> defined @Experiencer2/core.lua:1338
 PET_BATTLE_OPENING_START = <function> defined @Experiencer2/events.lua:21
 RefreshText = <function> defined @Experiencer2/core.lua:322
 defaultModuleLibraries = <table> {
 }
 FormatNumberFancy = <function> defined @Experiencer2/core.lua:286
 modulesInitialized = true
 GetNumOfEnabledModules = <function> defined @Experiencer2/core.lua:804
 UpdateDataBrokerText = <function> defined @Experiencer2/databroker.lua:29
 ShouldShowSecondaryText = <function> defined @Experiencer2/core.lua:591
 GetAnchors = <function> defined @
```

**Comment by DJScias** [comment]
I'm currently rewriting the addon for TWW prepatch (#6).

Considering I touch upon the parts that create this issue, I'll see if I can fix it alongside that.

**Comment by DJScias** [comment]
Should be fixed in 6.0.0 (via 0632b22b8755c651498b157a640af2830d6a8ddb).

PIlease do let me know if you do ever run into it again.
